YoutubeDownloader/YoutubeDownloader.csproj

```diff
@@ -2,27 +2,45 @@
 
   <PropertyGroup>
     <OutputType>WinExe</OutputType>
```

Comment:
Should this be changed or does it need to be `WinExe` still?

```suggestion
    <OutputType>Exe</OutputType>
```

Reply:
Yes, WinExe is still correct cross platform, if it was changed to Exe a console window would be started as well